packages/lit-dev-tools-cjs/src/api-docs/configs/lit-2.ts

```diff
@@ -20,7 +20,7 @@ const srcDir = pathlib.join(litDir, 'src');
  */
 export const lit2Config: ApiDocsConfig = {
   repo: 'https://github.com/lit/lit',
-  commit: 'f8ee010bc515e4bb319e98408d38ef3d971cc08b',
+  commit: '8c4c845b016bdbe93bf644d0160415d011166faa',
```

Comment:
This is why I might prefer to see release tags used instead of SHAs. I would have thought this would be commit for the lit@2.0.2 release: https://github.com/lit/lit/releases/tag/lit%402.0.2 so it took a bit to realize this was the for the lit-html@2.0.2 release. Of course release tags in the monorepo are going to have a somewhat similar question of package release was the most recent...

Reply:
Yeah, we rarely release the lit package, so using the lit release tag would mean we'll lag behind on most of the docs updates, which are mostly in the other packages.

Reply:
We could just pick any tag from the most recent set of releases each time. That would work, though the string is included in the "view source" links we generate, so you might be clicking on "view source" for a thing in one package, and see the name of another package in the URL, which could be slightly confusing (e.g. you click on LitElement but the URL has @lit/localize in it).

Reply:
In this case we really want the most recent release tag/SHA of `lit`, `lit-html`, `lit-element`, or `@lit/reactive-element`, right? I guess a comment saying that might make it pretty clear what any updater should do here, and we could keep it a SHA.

Reply:
Ok, PTAL. I used the latest lit-html tag and added a comment.

"view source" URLs now look like this: https://github.com/lit/lit/blob/lit-html@2.0.2/packages/reactive-element/src/reactive-element.ts#L801

We could also automatically convert the tag into a SHA for these URLs, if we don't want the slightly odd arbitrary package tag appearing instead of a SHA. WDYT?

Reply:
That's in some ways a much nicer URL, but in others it's a little weird because you're looking at reactive element with the `lit-html@2.0.2` tag. 🤷‍♂️ 

My brain is going to more complicated schemes where the URL is based on the package and its latest tag, but I don't think that's actually a good idea.

Reply:
Yeah, that wouldn't be good because the re-exporting packages would point to older SHAs than newer releases of the packages they re-export. I suspect a SHA here would be less confusing because it's opaque, or we could come up with a date-or sequence-based immutable tag we use every time we release. SHA works now though and unblocks this PR :)

Reply:
Switched back to the SHA!